docker/postgres/docker-entrypoint-initdb.d/init.0.sql

```diff
@@ -0,0 +1,251 @@
+-- We begin a transaction so that if any SQL statement fails, none of the
+-- changes will be applied.
+begin;
```

Comment:
Is it possible to keep this file in `examples/forum/schema.sql` instead of copying it here? Or maybe it could be a good idea to add an `examples/docker` folder that uses this build.